src/api/app.js

```diff
@@ -44,14 +44,17 @@ module.exports.manifest = {
  *       console.log('SAFEApp instance initialised and handle returned: ', appHandle);
  *    });
  **/
-module.exports._with_async_cb_initialise = (appInfo, safeAppGroupId) => {
+
+// QUESTION: How does user obtain safeAppGroupId?
```

Comment:
The user is not expected to obtain that value, that's internally used by the browser and used to call the function to free the objects when refreshing or closing a tab. Please remove this comments from the code.